pkgs/applications/version-management/p4/default.nix

```diff
@@ -7,7 +7,7 @@ stdenv.mkDerivation rec {
   src = fetchurl {
     # actually  https://cdist2.perforce.com/perforce/r21.2/bin.linux26x86_64/helix-core-server.tgz but upstream deletes releases
     url = "https://web.archive.org/web/20211118024943/https://cdist2.perforce.com/perforce/r21.2/bin.linux26x86_64/helix-core-server.tgz";
-    sha256 = "sha256-cmIMVek4lwVYJQbW8ziABftPZ0iIoAoSpR7cKuN4I7M=";
```

Comment:
I think we need to re-upload to archive.org instead because upstream might keep breaking this.

Reply:
This is what I tried here: https://github.com/NixOS/nixpkgs/commit/446701c15529af76c28a3fe1dd21a7ebd82f43af#diff-e8a54ba497dd42c5b2fd3dcc8ee23bb21f4adadf937040970b6f29225d9b8890R9

Reply:
cc @corngood

Reply:
I don't understand why that would be needed.  A timestamped link on archive.org should be immutable.  Upstream shouldn't be able to break it.  I think maybe the hash was just wrong on the previous commit?